<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WicdKcmWidget</class>
 <widget class="QWidget" name="WicdKcmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>588</width>
    <height>526</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="KTabWidget" name="ktabwidget">
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Network interfaces</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QFormLayout" name="formLayout_7">
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Wireless interface:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Wired interface:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KComboBox" name="wiredBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KComboBox" name="wirelessBox">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="frame">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="showWiredBox">
            <property name="text">
             <string>Always show wired interface</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="swithcWiredBox">
            <property name="text">
             <string>Always switch to a wired connection when available</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Global DNS servers</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QCheckBox" name="useDNSBox">
            <property name="text">
             <string>Use global DNS servers</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QFormLayout" name="formLayout_2">
            <property name="fieldGrowthPolicy">
             <enum>QFormLayout::ExpandingFieldsGrow</enum>
            </property>
            <property name="labelAlignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
            <item row="1" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>DNS domain:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="KLineEdit" name="domainEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Search domain:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="KLineEdit" name="searchDomainEdit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>DNS server 1:</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="KLineEdit" name="server1Edit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>DNS server 2:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1">
             <widget class="KLineEdit" name="server2Edit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="5" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>DNS server 3:</string>
              </property>
             </widget>
            </item>
            <item row="5" column="1">
             <widget class="KLineEdit" name="server3Edit">
              <property name="enabled">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Other</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Wired automatic connection:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="KComboBox" name="automaticWiredBox"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="reconnectBox">
            <property name="text">
             <string>Automatically reconnect on network connection loss</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>External programs</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout_3">
       <property name="fieldGrowthPolicy">
        <enum>QFormLayout::ExpandingFieldsGrow</enum>
       </property>
       <item row="1" column="0">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Route table flushing:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="KComboBox" name="routeBox"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Wired link detection:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="KComboBox" name="wiredLinkBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>DHCP client:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="KComboBox" name="dhcpBox"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Advanced settings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QGroupBox" name="groupBox_5">
         <property name="title">
          <string>Miscellaneous</string>
         </property>
         <layout class="QFormLayout" name="formLayout">
          <property name="fieldGrowthPolicy">
           <enum>QFormLayout::ExpandingFieldsGrow</enum>
          </property>
          <property name="labelAlignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_15">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>WPA Supplicant driver:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="KComboBox" name="wpaSupplicantBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_16">
            <property name="minimumSize">
             <size>
              <width>80</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Backend:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="KComboBox" name="backendBox">
            <property name="minimumSize">
             <size>
              <width>120</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_7">
         <property name="title">
          <string>Debugging</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <widget class="QCheckBox" name="debugBox">
            <property name="text">
             <string>Enable debug mode</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_8">
         <property name="title">
          <string>Wireless interface</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QCheckBox" name="dbmBox">
            <property name="text">
             <string>Use dBm to measure signal strength</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Ping static gateways after connecting to verify association</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KComboBox</class>
   <extends>QComboBox</extends>
   <header>kcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
  <customwidget>
   <class>KTabWidget</class>
   <extends>QTabWidget</extends>
   <header>ktabwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>domainEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>252</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>searchDomainEdit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server1Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>308</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server2Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>useDNSBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>server3Edit</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>293</x>
     <y>225</y>
    </hint>
    <hint type="destinationlabel">
     <x>331</x>
     <y>364</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
